app/models/amr_count.rb

```diff
@@ -0,0 +1,14 @@
+class AmrCount < ApplicationRecord
+  belongs_to :pipeline_run
+  def self.viewable(user)
+    if user.admin?
+      all
+    end
```

Comment:
so the functions here are not implemented?

Reply:
If you are not calling these functions, I don't think you need to implement them here.